Observer/OrderSaveAfter.php

```diff
@@ -1,5 +1,6 @@
 <?php
 namespace Stock2Shop\OrderExport\Observer;
+use Stock2Shop\OrderExport\Observer\OrderLineError as OLE;
```

Comment:
Can we rather just keep this as OrderLineError 

Not sure the shortening is necessary (and is confusing)

Reply:
[452dfef](https://github.com/stock2shop/magento2_module_webhook/pull/12/commits/452dfefcd283ddcef5b046ae70d9c6aa8ec20097)